File: 171-Excel Sheet Column Number/171.py
class Solution(object):
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        ans = 0
        for i, j in zip(range(len(s)-1, -1, -1), range(len(s))):
            ans += (ord(s[i]) - ord('A') + 1) * pow(26, j)
        return ans

File: 171. Excel Sheet Column Number/171.cpp
class Solution {
public:
    int titleToNumber(string s) {
        int ans = 0;
        std::reverse(s.begin(), s.end());
        for(int i=0; i<s.length(); ++i)
            ans += (s[i]-'A'+1) * pow(26, i);
        return ans;
    }
};